README.rst

```diff
@@ -29,6 +29,8 @@ To get started using the library, here's the full usage:
     >>> client.stop_motion(bedNumber, side)
     >>> client.stop_pump(bedNumber)
     >>> pprint(client.foundation_status(bedNumber))
+    >>> pprint(foundation_system(bedNumber))
```

Comment:
Should these be `client.foundation_system(bedNumber)`?

Reply:
yes, fixed